censusreporter/apps/census/static/js/charts.js

```diff
@@ -1181,7 +1182,7 @@ function Chart(options) {
             });
         }
     }
-    
+
     // format percentages and/or dollar signs
     chart.valFmt = function(value, decimals) {
         var precision = (!!decimals) ? decimals : 0,
```

Comment:
What if we changed this ``0`` to ``chart.decimalPlaces`` -- the chart object is specific to the current chart so using its default decimal places makes sense.

Then we wouldn't need to make any other changes at all, except to inject that value in the chart's constructor.